client/data/sites/site-excerpt-constants.ts

```diff
@@ -5,6 +5,7 @@ export const SITE_EXCERPT_REQUEST_FIELDS = [
 	'URL',
 	'is_coming_soon',
 	'is_private',
+	'visible',
```

Comment:
I try to understand why we need that - that data, including the `visible` key, is already available in the `sites` store. What is the use case in which we need this?

Reply:
> that data, including the `visible` key, is already available in the `sites` store. What is the use case in which we need this?

If the `sites` store isn't yet loaded, we use `useSiteExcerptsQuery` instead. This `SITE_EXCERPT_REQUEST_FIELDS` constant is used to determine the fields loaded by `useSiteExcerptsQuery`.